hera_mc/geo_handling.py

```diff
@@ -321,7 +321,9 @@ def get_ants_installed_since(self, query_date, station_types_to_check='all'):
             if a.station_type_name.lower() in station_types_to_check:
                 found_stations.append(a)
                 if self.fp_out is not None and not self.testing:
-                    self.fp_out.write('{:6} {:.2f} {:.2f} {:.4f} {:.4f}\n'.format(station_name, a.easting, a.northing, a.lon, a.lat))
+                    self.fp_out.write(
+                        '{:6} {:.2f} {:.2f} {:.4f} {:.4f}\n'
+                        .format(a.station_name, a.easting, a.northing, a.lon, a.lat))
```

Comment:
I think this was an actual bug (`station_name` was never defined). I made a guess at how to fix it, but since it wasn't caught by tests before, I'm not confident in the fix. @david-deboer can you check that this is right and figure out why it wasn't causing tests to fail before?

Reply:
Testing didn't catch it because the write fp_out was not set.  Your fix would fix it, but I actually tend to think we should remove the file write option (but that should be a different PR).  For now, let's go with your fix and I'll look into a future PR.

Reply:
do you want to make an issue?

Reply:
Made issue